integration/internal/network/ops.go

```diff
@@ -45,6 +45,19 @@ func WithMacvlan(parent string) func(*types.NetworkCreate) {
 	}
 }
 
+// WithMacvlanPassthru sets the network as macvlan with the specified parent in passthru mode
+func WithMacvlanPassthru(parent string) func(*types.NetworkCreate) {
```

Comment:
Looks like this no longer compiles when merged into master; I noticed this was failing when I restarted  Jenkins (which does a merge before testing);

[2024-06-15T13:02:18.893Z] # github.com/docker/docker/integration/internal/network
[2024-06-15T13:02:18.893Z] integration/internal/network/ops.go:49:47: undefined: types
[2024-06-15T13:02:18.893Z] integration/internal/network/ops.go:50:17: undefined: types


This needs to be updated to use `network.CreateOptions`.


Let me do a quick rebase, and fix this so that Ci can do another run 👍